Dockerfile

```diff
@@ -1,12 +1,12 @@
-FROM codeclimate/alpine-ruby:b38
+FROM ruby:2.3-alpine
```

Comment:
Our spec says [512 MB](https://github.com/codeclimate/spec/blob/master/SPEC.md#resource-restrictions) so this should be ok!